common/junit-platform-native/build.gradle

```diff
@@ -4,6 +4,7 @@ plugins {
     id 'maven-publish'
     id 'signing'
     id 'org.graalvm.build.interop.publishing'
+    id 'eclipse'
```

Comment:
I tend to shy away from Buildship, because I have always run into issues with it.

If it's a problem for the team, feel free to remove it, and I'll deal with it some other way.